### DISCUSSION TITLE
Env file

### DISCUSSION THREAD

**Discussion by muzammal-murtaza** [chatwoot/chatwoot#1820] [Self hosted] [answered]
I am getting error while login with facebook, i think env file is missing, can i get env file?

**Comment by sojan-official** [comment, marked as answer]
@muzammal-murtaza  if you on a self-hosted installation. please follow the instructions to set up Facebook 

https://www.chatwoot.com/docs/facebook-setup

**Comment by pranavrajs** [comment]
Moving to discussions as this is not an issue that needs to be tracked.
